translator/translate/logs/metrics_collected/prometheus/ruleDropUnsupportMetric.go

```diff
@@ -0,0 +1,30 @@
+package emfprocessor
+
+import (
+	"github.com/aws/amazon-cloudwatch-agent/translator"
+)
+
+const (
+	SectionKeyDropUnsupportedMetric = "drop_unsupported_metric"
+	DefaultDropUnsupportedMetric = true
+)
+
+type DropUnsupportedMetric struct {
+}
+
+func (d *DropUnsupportedMetric) ApplyRule(input interface{}) (returnKey string, returnVal interface{}) {
```

Comment:
Can you add a unit test for this?

Reply:
The schema test is a unit test for [this rule](https://github.com/aws/amazon-cloudwatch-agent/pull/407/files#diff-cc5c0c08c341aac1f4d061da8ae056f92d2d6c01e4e4286d693e95b8068eef72R21). Meaning after translating the CWAgent's configuration file to toml file with the predefined rules, it would validate between the expected toml file and the actual one.